src/styles/themes/libris.css

```
@@ -0,0 +1,996 @@
+        from
+          color-mix(
+            /* srgb actually gives a better optiucacl mix than oklch or hsl - we
+            *don't* want a hue between the paper's beige and the alert color,
+            because that would yield e.g. green when the alert is blue. */
+              in srgb,
+            var(--color-base-200),
+            var(--libris-alert-color, var(--color-base-200)) 20%
+          )
+          0.9 c h
+      );
+      /*background-color: oklch(from var(--libris-alert-color) 0.95 0.03 h);*/
```

Stylelint (17.14.0)

[error] Expected whitespace after "/*" (comment-whitespace-inside)

(comment-whitespace-inside)